browse/services/database/models.py

```diff
@@ -213,10 +216,30 @@ class User(db.Model):
                              index=True, server_default=text("''"))
     flag_allow_tex_produced = Column(
         Integer, nullable=False, server_default=text("'0'"))
-
+    flag_can_lock = Column(Integer,
+                           nullable=False, server_default=text("'0'"))
     tapir_policy_class = relationship('UserPolicyClass')
 
 
+
+class Nickname(db.Model):
+    __tablename__ = 'tapir_nicknames'
+    __table_args__ = (
+        Index('user_id', 'user_id', 'user_seq', unique=True),
+    )
+  
+    nick_id = Column(Integer, primary_key=True)
+    nickname = Column(String(20), nullable=False, unique=True, server_default=text("''"))
+    user_id = Column(ForeignKey('tapir_users.user_id'), nullable=False, server_default=text("'0'"))
+    user_seq = Column(Integer, nullable=False, server_default=text("'0'"))
+    flag_valid = Column(Integer, nullable=False, index=True, server_default=text("'0'"))
+    role = Column(Integer, nullable=False, index=True, server_default=text("'0'"))
+    policy = Column(Integer, nullable=False, index=True, server_default=text("'0'"))
+    flag_primary = Column(Integer, nullable=False, server_default=text("'0'"))
+
+    user = relationship('User')
+
+    
 class UserPolicyClass(db.Model):
```

Comment:
This isn't part of this PR's changeset but I can't remember why we needed this in browse.

Reply:
It is part of the foreign keys for one of the other tables.